scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
```

Comment:
Can you add some more color here...

* Jumbo frames are 6x the size of typical ethernet frames.
* Jumbo frames reduce the number of interrupts by the same factor (6x), and thus will reduce CPU utilization on both the send and receive sides of a connection.
* This larger frame size will increase network latency due to the properties of the Nagle algorithm  https://en.wikipedia.org/wiki/Nagle%27s_algorithm (enabled by default on Linux), that require the network's minimum segment size to be queued before being dispatched.
* Nagle's algorithm can be disabled by opening a tcp socket with TCP_NODELAY socket option set.

/cc @danwinship @dcbw y'all comfortable with this additional info?